.github/workflows/build-ci.yml

```diff
@@ -27,7 +27,10 @@ jobs:
                     - php: "8.1"
                       mongodb: "5.0"
                       mode: "low-deps"
-
+                    # Laravel 11
+                    - php: "8.3"
+                      mongodb: "7.0"
+                      mode: "dev"
```

Comment:
Once Laravel 11 will be released, this will no longer be necessary. But only `low-deps` job will run Laravel 10.